pkg/execution/batch/lua/append.lua

```diff
@@ -5,13 +5,14 @@
 local batchPointerKey = KEYS[1]      -- key to the batch pointer
 
 local batchLimit = tonumber(ARGV[1]) -- max size configured for this batch
-local event = ARGV[2]                -- event to be appended to the batch
-local newULID = ARGV[3]              -- ULID to update the pointer with, either if the batch is full or doesn't exist
-local prefix = ARGV[4]               -- the prefix used for redis
+local eventID = ARGV[2]
+local event = ARGV[3]                -- event to be appended to the batch
```

Comment:
It looks like we're reordering positional args passed to `append.lua`. How does that coordinate with Go code changes? In other words, will we have a period of time where some Go is passing `event` as `ARGV[2]` and some is passing as `ARGV[3]`?

Reply:
The Lua scripts are embedded in Go, there is decoupling so the old services use the old scripts while the new services use the new scripts